.gitignore

```diff
@@ -24,6 +24,9 @@ yarn-error.log
 *.d.ts
 !packages/**/*/types/**/*
 !packages/**/index.d.ts
+!tools/**/*/types/**/*
+!tools/**/index.d.ts
+!tools/**/types.d.ts
```

Comment:
@tlouisse  I saw this file, not sure if it's even used but I didn't wanna throw it away. Can you maybe look into it or put it inside a types folder so that we can remove this line 29 :P because it's only that one file for which I added it.

Reply:
Please don't throw it away indeed :) It is used, but Providence is definitele not fully typed.
Feel free to move it around and update references or do you want me to have a look at it?

Reply:
Naah it's okay for now, just as long as we have "some" plan moving forward (type providence, move this file into types folder so we dont need more .<foo>ignore entries)